### DISCUSSION TITLE
Test saleor core

### DISCUSSION THREAD

**Discussion by antoniosobrinho** [saleor/saleor#6616] [General]
I use saleor 2.11 and try run tests. I use `python manager.py test --settings=saleor.tests.settings` but got  `ERROR: usage: manage.py [options] [file_or_dir] [file_or_dir] [...]
manage.py: error: unrecognized arguments: -n --vcr-record-mode=none --ds=saleor.tests.settings`. Someone can help me?

**Comment by IKarbowiak** [comment]
Saleor is using pytest so you just type `pytest` in your console in the directory with `manage.py` file. Also, it's recommended to use `--reuse-db` flag to reuse the database between test runs and faster them.
